docassemble/AssemblyLine/data/static/al_audio.js

```diff
@@ -1,74 +1,98 @@
+// Do not overwrite already created al_js object. This may be relevant as our js gets built out.
+if ( !al_js ) { var al_js = {}; }
+
 $(document).on('daPageLoad', function() {
-  var $audio_nodes = $('.daaudio-control');
-  var id_count = 1;
-  for ( var audio_node of $audio_nodes ) {
-    al_js.replace_with_audio_minimal_controls( audio_node, 'page_reader_substitute_' + id_count );
-    id_count++;
-  }
+  try {
+    var $audio_nodes = $('.daaudio-control');
+    var id_count = 1;
+    for ( var audio_node of $audio_nodes ) {
+      al_js.replace_with_audio_minimal_controls( audio_node, 'page_reader_substitute_' + id_count, $audio_nodes );
```

Comment:
You've added an extra function param here, but not on line 19 where you define the function itself. Is there some JS trick going on that I don't understand?

Reply:
Good eye. That is a remnant of another approach I left behind. Thanks for the find!